### ISSUE TITLE
DNA binding site motif in the browser

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1654]
replaces
https://github.com/pombase/website/issues/903
https://github.com/pombase/website/issues/1419
https://github.com/pombase/website/issues/802

These are  provided by a script that uses these binding sites
https://www.pombase.org/browse-curation/dna-binding-sites
The script needs rerunning to pick up new binding sites.

Suggested additional changes:

1. [x] Ask pombelist if they know of any additional (and find a more prominent place to host the list)
users seem to be quite interested in this data
https://user-images.githubusercontent.com/7359272/41833353-e2468ef8-7847-11e8-9a46-c5aba5bd2186.jpg

2. [x] From @Antonialock  extend page using Maria's data (I don't know which data this means?)

3. [x]  Standardize the motif syntax   on webpage https://www.pombase.org/browse-curation/dna-binding-sites
    remove [5'-
    Is the symbol for OR a "/" (sometimes a | is used)

4. [x]  Omit mating-type M-box ACAAT, there is only one functional version of this, it depends on context. Currently, we flag  M-box ACAAT 45439 feature.

5. [x]  CCAAT box currently uses consensus CAAT so it is over-represented. Check this is updated

6. [x]   Prepare one track for each TF binding motif?
using https://github.com/pombase/website/blob/master/src/docs/browse-curation/dna-binding-sites.PomBase.md
https://github.com/pombase/legacy-eg-loader
https://github.com/pombase/legacy-eg-loader/blob/master/scripts/GenerateBindingMotifTrack.sh

7. [x]  If we don't have separate tracks, we need pop-up display labels:
WGATAA display label would be "iron repressed GATA element"
ACAAT display label would be mating type M-box
(permanent labels would be too cluttered)
although I think separate tracks is preferable

**Comment by ValWood** [comment]
8. Add labels to the features
9. Create one file per TF, and one composite file

**Comment by kimrutherford** [comment]
> Prepare one track for each TF binding motif?
> using https://github.com/pombase/website/blob/master/src/docs/browse-curation/dna-binding-sites.PomBase.md
> https://github.com/pombase/legacy-eg-loader
> https://github.com/pombase/legacy-eg-loader/blob/master/scripts/GenerateBindingMotifTrack.sh

Reminder to myself, the Python code is here:
 - https://github.com/pombase/legacy-eg-loader/blob/master/misc_scripts/GenerateBindingSitTracks.py

**Comment by PCarme** [comment]
@ValWood Should I merge all of the instances where I have found some level of redundancy, or only the ones where there is a complete overlap ?

**Comment by ValWood** [comment]
Good question. I thought it would be complicated.

If one is a subset of the other, do you think it is better to use the more restrictive, or permissive? I wasn't sure.
Maybe the most permissive (which will give more false P) , but noting the  the more restrictive version, in case we dicide to change?

Have you got any where they completely diverge ? if so, I think have  x-1 and x2 or incorporate the relevant ambiguity codes

**Comment by PCarme** [comment]
Yes I have a few examples where the same TF has 2 completely different sequences. Sometimes one consensus is the reverse complement of the other sequence, sometimes there is 1 or 2 nt of difference (for these ones I can try and adapt the ambiguity codes to reflect both consensus and put both sources in the last column).

I'll have a closer look at which ones are going to be complicated, I can already go ahead and merge the simple ones

**Comment by ValWood** [comment]
Convert 
https://www.pombase.org/browse-curation/dna-binding-sites
into a browser track per motif

**Comment by kimrutherford** [comment]
> Inr consensus initiator sequence 	PyPyPuN(A/C)(C/A)
>
> `(A/C)(C/A)`

That looks a bit weird.  Maybe there is a typo?

**Comment by kimrutherford** [comment]
> It would be very cool to link each consensus sequence to the specific tracks from this page

So link to JBrowse from table?

**Comment by ValWood** [comment]
>So link to JBrowse from table?

Yes, so people could immediately go from here to a specific track without needing to locate it, which can take a while.  God, I do mollycoddle them...

**Comment by kimrutherford** [comment]
> I had a look at the corresponding track and this consensus is, indeed, everywhere in the genome...

Is that track useful?  Should we remove it?

**Comment by kimrutherford** [comment]
> I think the (A/C)(C/A) part is to account for which of the 2 nucleotides is the most common (A is the most common but it can also be a C in the first position, C is the most common but in can also be an A in the second position)

That makes sense.  Thanks for checking.

**Comment by kimrutherford** [comment]
> It is described as PyPyPuN in the linked article, but maybe we should standardise using the ambiguity codes for all the sequences on this page ?

I think we should.  We have links to the papers for anyone who wants to see the original motif.

**Comment by kimrutherford** [comment]
The new script is here:
https://github.com/pombase/pombase-scripts/blob/main/generate_binding_sites_track.py

The how-to docs are here:
https://github.com/pombase/website/wiki/Formatting-data-files-for-JBrowse#creating-tf-binding-site-tracks

I think we can close this and open new issues for fixes and updates.

**Comment by ValWood** [comment]
Brill, this update will be very popular, I'm sure!

**Comment by ValWood** [comment]
The graph would be quite nice to post with the announcement and/or news item
